.github/workflows/build.yml

```diff
@@ -25,11 +25,18 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python ${{ matrix.python-version }} for PR
+      if: github.event_name != 'push'
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Set up old Python for PyPI
+      if: github.event_name == 'push'
```

Comment:
need to use python 3.4 during Release on linux for compatible "manylinux" PyPI (user's python must be newer/same)